src/sparseml/pytorch/__init__.py

```diff
@@ -16,6 +16,29 @@
 Functionality for working with and sparsifying Models in the PyTorch framework
 """
 
+import os
+
+from packaging import version
+
+
+try:
+    import torch
+
+    _PARSED_TORCH_VERSION = version.parse(torch.__version__)
+    if (
+        _PARSED_TORCH_VERSION.major == 1
+        and _PARSED_TORCH_VERSION.minor in [10, 11]
+        and os.environ.get("NM_BYPASS_TORCH_VERSION", "").lower() == "true"
```

Comment:
shouldn't his be a not?
also in general for environment variables, we use integers for env var flags (ie 1/0)
```suggestion
        and not os.environ.get("NM_BYPASS_TORCH_VERSION", "")
```

Reply:
Ahh okay I can change to int. `not` won't work because any non-empty string will resolve to True. so "0" would be True.